src/app/modules/material.module.ts

```diff
@@ -0,0 +1,8 @@
+import { NgModule } from '@angular/core';
+import {MdInputModule, MdDatepickerModule, MdCoreModule, MdNativeDateModule, MdExpansionModule, MdSelectModule} from '@angular/material';
+
+@NgModule({
+  imports: [MdInputModule, MdDatepickerModule, MdCoreModule, MdNativeDateModule, MdExpansionModule, MdSelectModule],
```

Comment:
It is more readable to have this modules in column instead of row (just a recommendation)